### DISCUSSION TITLE
OC4 Environmental Loads

### DISCUSSION THREAD

**Discussion by borjabenitez** [OpenFAST/openfast#1520] [General]
Dear all,

For the OC4 validation, I would like to know more in depth the environmental loads that have been used, especially the parameters of the Weibull distribution for wave and wind. 

Thanks.

**Comment by jjonkman** [comment]
Dear @borjabenitez,

Are you referring to the IEA Wind Task 30 OC5 Phase II project regarding validation of the DeepCwind floating semisubmersible based on data from the offshore wave basin of MARIN or something else?

Best regards,

**Comment by borjabenitez** [comment]
Dear @jjonkman

I was referring to OC4, phase I for offshore wind turbine with jacket support structure.

Best regards.

**Reply by jjonkman** [reply]
Dear @borjabenitez,

IEA Wind Task 30 OC4 Phase 1 considered a specific set of load cases, each with distinct wind and waves, but did not consider a specific probability distribution of wind or wave conditions. 

Best regards,

**Reply by borjabenitez** [reply]
Thank you very much @jjonkman 

Best regards.
